vector_double.cpp

```diff
@@ -0,0 +1,207 @@
+#include "vector_double.hpp"
+
+
+template <typename T>
+Vector<T>::Vector()
+{
+    size_v = 0;
+    space  = 0;
+    elem   = nullptr;
+
+}
+
+
+// Copy Constructor
+template<class T>
+Vector<T>::Vector(const Vector<T>& b)
+{
+    space = b.space;
+    size_v = b.size_v;
+    elem = b.elem;
```

Comment:
this doesn't work for a copy constructor -- we need to preserve `b`'s elem pointer and copy its data into `this`

Reply:
Again, use member initialization lists for constructors if not doing anything but initializing.